src/WinRT.Runtime/Interop/IActivationFactory.cs

```diff
@@ -24,9 +24,25 @@ interface IActivationFactory
 
 namespace ABI.WinRT.Interop
 {
+#if EMBED 
+    internal
+#else
+    public
+#endif
+    static class IActivationFactoryMethods
+    {
+#if NET
+        public static global::System.Guid IID { get; } = new Guid(new global::System.ReadOnlySpan<byte>(new byte[] { 0x35, 0, 0, 0, 0, 0, 0, 0, 0xC0, 0, 0, 0, 0, 0, 0, 0x46 }));
+#else
+        public static global::System.Guid IID { get; } = new(0x00000035, 0, 0, 0xC0, 0, 0, 0, 0, 0, 0, 0x46);
+#endif

```

Comment:
I did something similar in InterfaceIIDs. Do we want to remove those?

Reply:
Possibly, will handle cleanup in separate PR.